mcs/class/corlib/corlib_xtest.dll.sources

```diff
@@ -219,14 +219,12 @@
 ../../../external/corefx/src/Common/tests/System/ObjectCloner.cs
```

Comment:
there are more listed above which are in System.Core now:

```
../../../external/corefx/src/Common/tests/System/Collections/CollectionAsserts.cs
../../../external/corefx/src/Common/tests/System/Collections/TestBase.Generic.cs
../../../external/corefx/src/Common/tests/System/Collections/TestBase.NonGeneric.cs
../../../external/corefx/src/Common/tests/System/Collections/ICollection.Generic.Tests.cs
../../../external/corefx/src/Common/tests/System/Collections/ISet.Generic.Tests.cs
../../../external/corefx/src/Common/tests/System/Collections/IEnumerable.NonGeneric.Tests.cs
../../../external/corefx/src/Common/tests/System/Collections/IEnumerable.Generic.Tests.cs
```

Reply:
That's correct these are abstract types with data or common fixtures